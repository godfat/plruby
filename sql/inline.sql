DO $$ p('INLINE') $$ LANGUAGE plruby;
