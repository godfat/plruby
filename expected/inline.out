DO $$ p('INLINE') $$ LANGUAGE plruby;
NOTICE:   p('INLINE') 
